build_docs.py

```diff
@@ -1053,28 +1003,60 @@ def purge_path(www_root: Path, path: Path):
     run(["curl", "-XPURGE", f"https://docs.python.org/{{{','.join(to_purge)}}}"])
 
 
+def parse_versions_from_devguide():
+    releases = requests.get(
+        "https://raw.githubusercontent.com/"
+        "python/devguide/main/include/release-cycle.json",
+        timeout=30,
+    ).json()
+    return [Version.from_json(name, release) for name, release in releases.items()]
+
+
+def parse_languages_from_config():
+    """Read config.toml to discover languages to build."""
+    config = tomlkit.parse((HERE / "config.toml").read_text(encoding="UTF-8"))
+    languages = []
+    for name, section in config["languages"].items():
+        languages.append(
+            Language(
+                name,
```

Comment:
Could you do 
```suggestion
                name.replace('_', '-').lower(),
```

Here and then put e.g. pt_BR as the section name, avoiding having to special case the ISO tag as an optional variable?

Reply:
Yes, nice, done, thanks!!